### DISCUSSION TITLE
Monster versions

### DISCUSSION THREAD

**Discussion by SummerSky777** [decheine/complete-compendium#225] [General]
Some official material have some errors, by omission and some forgetfulness it seems. 
 
  The snow golem https://www.completecompendium.com/appendix/golervsn/ has two versions in the books, but they are not named as such. Lesser and Greater, or proto-snow golem and just snow golem. 
   The one in the adventure is different than the one in the appendix. It first appears in the adventure.
[Ravenloft Appendix III (2153)](https://www.completecompendium.com/catalog/rv/2153/)
[RA2 Ship of Horrors (9321)](https://www.completecompendium.com/catalog/rv/9321/)

The Ravenloft Appendix 3 stats of the Snow Golem (greater snow golem),
XP value of 7,000
AC 1
2 attacks per round
releases a cloud of scalding steam when harmed by magical fire
sunders weapons when struck

The Ship of Horror stats of the Snow Golem (lesser snow golem)
XP value of 6,000
AC 6
1 attack per round
  
Also the Appendix 3 golem is described as "First discovered on the arctic island of Todstein (as described in the adventure Ship of Horror), snow [golems](https://www.completecompendium.com/appendix/golem) are similar to [greater golems](https://www.completecompendium.com/appendix/golem1). Like others of their ilk, they are usually employed to guard an important item or entryway."
While the one from Ship of Horror as "A snow golem is a weak version of a greater golem. It is created in the same manner as any other golem"

So the ones in the island seems to be the prototypes. 

And if one makes the mistake of using the Appendix 3 Snow Golem statistics instead of the ones in the adventure, by some forgetfulness, then the characters will not even get to Meredoth. Which can kill them anyways since he is a 20th level necromancer in an adventure for 8-10th level characters.
The greater version of the snow golem destroys magical weapons and is much more dangerous in general.
"Due to the extreme cold of their bodies, nonmagical weapons made of metal shatter when used against a snow golem. Even magical arms are not immune to this effect, having a 35% chance of being destroyed with each successful blow. This chance is reduced by 5% for every plus of the weapon above +2 (i.e. a +3 weapon has only a 30% chance of shattering)."

noted in https://www.enworld.org/threads/2e-snow-golem-ravenloft-lesser-and-greater.703181/

**Comment by decheine** [comment]
Amazing find, thank you for this! I'll be sure to create unique pages for each.

**Comment by SummerSky777** [comment]
I don't know how many similar, or other types of cases, exist. Probably very few.

**Comment by SummerSky777** [comment]
Mind Flayer Vampire. This is and is not an error, and the only way to know the specifics is if we could walk to the man who designed the creature. But its so extreme that should be mentioned. Its from Thoughts of Darkness, an adventure meant to kill the players that was written like that with no doubt on purpose. 
https://www.completecompendium.com/appendix/vampilli/
https://www.completecompendium.com/catalog/rv/9364/

"Their powerful negative energy essence allows them to drain two life energy levels from anyone they strike."
"each of its four tentacles each round, inflicting 1d6+4 points of crushing damage per hit, in addition to its life energy drain."

So potentially 8 levels per round.

Cannot do anything about this, no errata was ever out.  Its just considered an extreme monster (extremely badly written). The adventure is not playable but has great art and some domain ideas, its 12-15 level but you need to be in a 30 level party of avatars to make it.

This is one of the most extreme monsters.

**Reply by decheine** [reply]
Oh yes, those guys are nasty, especially with that 90% magic resistance. I've run them before, but the party was 18th level. I made very clear how extremely dangerous they are and to never be in melee range if possible. I used the special vampiric illithid, the athaekeetha, as an "unkillable" boss that endlessly regenerates no matter what and said "good luck". my players solution made some of the best dnd I've ever seen: they trapped it in a pearl of power (after several attempts until it failed its MR and save) and then threw that into a bag of holding, destroying both items and hurling the creature into the astral plane.

the trick with regular mind flayers is lots of magic arrows and terrain manipulation. if you can keep them away from you they can be defeated, and you only need +1 weapon to hit. they are indeed one of the most extreme monsters I agree, but that's kind of the name of the game in high level ad&d 2e.

**Reply by decheine** [reply]
I read some of the Thoughts of Darkness adventure, and it definitely looks very brutal and requires an experienced party, it is definitely doable, although still deadly. The adventure states that the party _must_ have a priest capable of casting spells like raise dead, resurrection, and I can't recall if the book mentions it but definitely restoration to restore life energy levels. And the book says each player's AC should be between 0 to -5 with maybe a warrior with even lower AC. I've seen AC -8 on a warrior of only like 10th level. The vampiric illithids have an adjusted thac0 of 8, which is nasty but not too crazy, so on average around half of the melee attacks will hit. A well balanced and equipped party of 5 is capable of taking 3 of them, although it is almost guaranteed the party will lose some life energy levels they can get them all back if the priest is at least level 13 and can cast restoration. What shocked me more was encounter 6 with 6 stone giants, 10 mind flayers and 10 drow mages, that's nuts! But doable for a worthy group of players. I might run this adventure some time, it looks like a fun challenge that might take a couple of attempts.

**Comment by SummerSky777** [comment]
There is also this "monster" in Thoughts of Darkness but does not have compendium page in the adventure.

Dream Encounter, page 10 / dream adversary
Dream Adversary (1/PC): AL CE; AC 0; MV 12; hp 40; THACO 10; #AT same as opponent; Dmg by weapon; SA nil; SD nil; MR 80%; SZ M; ML 18; XP 500 - these attack you before encounter 1
a mirror copy of the heroes fighting them before encounter 1 with the drow and duergar

**Reply by decheine** [reply]
This is a good edge case. I'm leaning more towards classifying this as an "Encounter" more than a "monster". It doesn't have a full statblock, and the surrounding body text is describing the encounter. Since it lacks both a full statblock and monster-style body paragraphs, I don't think it should be included in the compendium as it currently exists. In the future, however, I may look to tabulate all of the various minor statblocks like this one or encounters, but they would be presented in a different way.

**Comment by SummerSky777** [comment]
This is like those monsters or creatures appearing only in spells, like mordenkainen's hound etc but not in the compendiums as themselves as independent of the effect that generated them.

**Reply by decheine** [reply]
Pretty much, yeah. Unless there is a full monster page describing the monster. There are quite a few of those, where the monster page is a full monster and a spell to create it.

**Comment by SummerSky777** [comment]
Another odd thing, the lobster is absent from monster manuals as a monster name but in the giant crayfish description it is suggested they are the same. This makes someone searching never find a lobster monster. Maybe a link directing the term giant lobster to the giant crayfish entry is needed with some note (lobsters are much larger than crayfish among other things).

**Reply by decheine** [reply]
Hmm, maybe. The way it's currently done is that the Appendix has many (>7000) titles with lots of redundancy and alternate names. I could see just adding "Lobster, Giant": "crusgian", but I think that might be too misleading. Since there isn't a proper "lobster" monster across the entire edition, at least that we can find so far, I don't think we should include it like that. Another thing that supports this claim is the way the [crusgian](https://www.completecompendium.com/appendix/crusgian/) page describes "The crayfish is essentially a freshwater lobster..." and "the giant crayfish, like the lobster, ..." which is referring to a 'creature' in a way that implies it's distinct from this creature. So the conclusion is I think there should only be a "Lobster, Giant" title if one is described directly in a monster entry.

**Comment by SummerSky777** [comment]
I guess they did not want to put in the title "giant freshwater lobster"

**Comment by SummerSky777** [comment]
Will  you be releasing "Offline Compendium" versions every on Oct 29 or in random times or every year at XX-XX?

**Reply by decheine** [reply]
I was thinking of doing Offline Compendium releases after certain milestones are reached, like I'll probably do one after I finish the like 5 or so main TSR books, and then another after Dragon Magazines, and then eventually when it's all done I'll release the full version. And then periodically for fixes. I haven't made a good script to convert the HTML files to offline compatible yet, I just did it manually the first time, so that's another thing I need to do

**Reply by decheine** [reply]
I should mention that you can host your own local version of the entire site just by cloning it and running it. The offline version is for standalone HTML
